aten/src/ATen/native/Loss.cpp

```diff
@@ -148,4 +148,36 @@ Tensor poisson_nll_loss(const Tensor& input, const Tensor& target, const bool lo
 
     return apply_loss_reduction(loss, reduction);
 }
+
+Tensor l1_loss(const Tensor& input, const Tensor& target, int64_t reduction) {
```

Comment:
Previously, the C++ implementation of l1_loss did not broadcast. Now, it does, so we can remove the `broadcast_tensors` line from
https://github.com/pytorch/pytorch/blob/8fb756d3b25be6b8f56bbd0d7b52b1f1ed1e7119/torch/nn/functional.py#L2177-L2180